<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>880</width>
         <height>558</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QFrame" name="frame">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(179, 255, 255);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0" colspan="3" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QLabel" name="label">
             <property name="whatsThis">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 900 10pt &quot;Arial Black&quot;;</string>
             </property>
             <property name="text">
              <string>Виберіть рівень гри</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="pushButton_5">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 170, 255);
font: 900 10pt &quot;Arial Black&quot;;</string>
             </property>
             <property name="text">
              <string>Складний рівень</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="pushButton_4">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 170, 255);
font: 900 10pt &quot;Arial Black&quot;;</string>
             </property>
             <property name="text">
              <string>Середній рівень</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="pushButton">
             <property name="styleSheet">
              <string notr="true">font: 900 10pt &quot;Arial Black&quot;;
background-color: rgb(255, 170, 255);</string>
             </property>
             <property name="text">
              <string>Легкий рівень</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_2">
             <property name="styleSheet">
              <string notr="true">font: 900 10pt &quot;Arial Black&quot;;</string>
             </property>
             <property name="text">
              <string>Ваш найкращий результат:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="label_5">
             <property name="styleSheet">
              <string notr="true">font: 900 10pt &quot;Arial Black&quot;;</string>
             </property>
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_3">
             <property name="styleSheet">
              <string notr="true">font: 900 10pt &quot;Arial Black&quot;;</string>
             </property>
             <property name="text">
              <string>Ваш попередній бал:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="label_4">
             <property name="styleSheet">
              <string notr="true">font: 900 10pt &quot;Arial Black&quot;;</string>
             </property>
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QPushButton" name="pushButton_2">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 170, 255);
font: 900 10pt &quot;Arial Black&quot;;</string>
             </property>
             <property name="text">
              <string>Налаштування</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QPushButton" name="pushButton_6">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 170, 255);
font: 900 10pt &quot;Arial Black&quot;;</string>
             </property>
             <property name="text">
              <string>Наставлення</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QPushButton" name="pushButton_3">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 170, 255);
font: 900 10pt &quot;Arial Black&quot;;</string>
             </property>
             <property name="text">
              <string>Вихід</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>568</y>
    </hint>
    <hint type="destinationlabel">
     <x>558</x>
     <y>469</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
